### ISSUE TITLE
Is BHL available on Linux or Android OS?

### ISSUE THREAD

**Issue by M4RC0-NTH** [RAINBOW2000/BHL#4]
hi....is Linux-specific language already available?

 and how about a guide to porting it also to Linux and Android?

**Comment by stephen-cusi** [comment]
After my test, it seems that this is not the bhl source code, and the author may have abandoned the account

**Comment by M4RC0-NTH** [comment]
Yeah..i think too...maybe this is a gunmod version or xashcoop mod.

**Comment by stephen-cusi** [comment]
> Yeah..i think too...maybe this is a gunmod version or xashcoop mod.

Feeling cheated 😂
